scripts/sync-admin-users.rb

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
+require 'json'
+require 'yaml'
+require 'net/https'
+
+require File.expand_path("../lib/mail_credentials_helper", __FILE__)
+require File.expand_path("../lib/uaa_sync_admin_users", __FILE__)
+
+def get_uaa_target(api_url)
```

Comment:
This looks convoluted as [we already know the UAA endpoint](https://github.com/alphagov/paas-cf/blob/53086d5f5e01204ed423ba711b91b38ce1e7174f/terraform/cloudfoundry/dns-records.tf#L9-L15)
Could we expose it as terraform output and read it as env variable?